exist-core/src/main/java/org/exist/backup/FileSystemBackupDescriptor.java

```diff
@@ -73,6 +76,29 @@ public BackupDescriptor getChildBackupDescriptor(final String describedItem) {
         return bd;
     }
 
+    @Override
+    public List<BackupDescriptor> getChildBackupDescriptors() {
+        try {
+            try (final Stream<BackupDescriptor> entries = Files.list(descriptor.getParent())
```

Comment:
Would it make sense to add some logging for this step (assuming this could consume some time)

Reply:
I added as requested :-)